paramiko/file.py

```diff
@@ -93,39 +94,20 @@ def flush(self):
         self._wbuffer = BytesIO()
         return
 
-    if PY2:
-
-        def next(self):
-            """
-            Returns the next line from the input, or raises
-            ``StopIteration`` when EOF is hit.  Unlike Python file
-            objects, it's okay to mix calls to `next` and `readline`.
-
-            :raises: ``StopIteration`` -- when the end of the file is reached.
-
-            :returns: a line (`str`) read from the file.
-            """
-            line = self.readline()
-            if not line:
-                raise StopIteration
-            return line
-
-    else:
-
-        def __next__(self):
-            """
-            Returns the next line from the input, or raises ``StopIteration``
-            when EOF is hit.  Unlike python file objects, it's okay to mix
-            calls to `.next` and `.readline`.
+    def __next__(self):
+        """
+        Returns the next line from the input, or raises ``StopIteration``
+        when EOF is hit.  Unlike python file objects, it's okay to mix
+        calls to `.next` and `.readline`.
 
-            :raises: ``StopIteration`` -- when the end of the file is reached.
+        :raises: ``StopIteration`` -- when the end of the file is reached.
 
-            :returns: a line (`str`) read from the file.
-            """
-            line = self.readline()
-            if not line:
-                raise StopIteration
-            return line
+        :returns: a line (`str`) read from the file.
```

Comment:
`str` or `bytes`?

Reply:
Either-or actually (see `SFTPFile.readline`), I'll enhance this to make that clearer.